src/interaction.test.ts

```diff
@@ -16,16 +16,16 @@ import {
 } from "./interaction";
 
 describe("KeystoreInteraction", () => {
-
   class TestKeystoreInteraction extends KeystoreInteraction {
+    param;
 
-    constructor({param}) {
+    constructor({ param }) {
       super();
       this.param = param;
     }
 
     messages() {
-      const messages = super.messages();
+      const messages: any = super.messages();
```

Comment:
This should be something like this no?

```suggestion
      const messages: Record<string, string>[] = super.messages();
```

Reply:
it's actually a `Message[]` but `Message` is not public. It can be represented as `Record<string, any>[]`, but this is a unit test. I think unit tests should be robust enough to work without typing since they are in the realm of runtime. Setting it as `any` in a world with `unknown` and `any` types will allow a test to express possible returns values which could happen within a codebase which has incomplete type tests.

I'll change it to `unknown`.

Reply:
So, for some reason addressing changes have caused typescript error 4053 to start being thrown across the code. I've never encountered this error, but it requires that `Message` be imported to a module just so that class methods can use the inherited return type. I don't know why it wasn't being throw before, and I probably never have encountered it because it seems to be happening due to heavy reliance on inheritance.

So, I have to export `interaction.Message` type now, but this test typing still doesn't work because the `param` field on each object is not part of type `Message` and is only used as sort of a test-id.